experiments/swdt/README.md

```diff
@@ -0,0 +1,76 @@
+# Design document for the SWDT CLI. 
+
+This document is a reference for decisions regarding multi-OS support architecture. It is developed with Cobra/spf13 to conform to a stable namespace and different connection libraries to support remote command execution.
+
+Follow the [docs](../../docs) for more information.
+
+## Subcommands and namespace
+
+These are the current supported subcommands for the program:
+
+* `swdt setup`
+  * Initialize the node setup, enabling RDP, installing Choco and packages, etc.
+* `swdt kubernetes`
+  * Deploy Kubernetes binaries from the HTTP server indicated in the configuration.
```

Comment:
maybe include local files? My dev workflow is to build the binaries locally and then ssh them to the node

Reply:
still in doubt about it, mine as well so I can compile locally and copy over SSH (that's the easier case, and probably how it will shaped on next PR).

if at some point this becomes a controller it gets trickier to push the file (just though on a HTTP server for now)

Reply:
I can see it useful to curl and endpoint such as https://dl.k8s.io/v1.29.0/bin/windows/amd64/kubelet.exe as well

Reply:
Copying binaries built from source is one of major features for me too, as [per my dream](https://github.com/kubernetes-sigs/sig-windows-dev-tools/blob/8ce22f46c7c53b7b41b54758e6bd3727b4b29ee3/docs/quo-vadis-swdt.md?plain=1#L74-L75), but I'd get the machinery working based on downloads of latest releases, them take care of builds e.g. which could be configurable in the `config.yaml`